shared_model/backend/protobuf/commands/proto_add_asset_quantity.hpp

```diff
@@ -57,28 +55,20 @@ namespace shared_model {
       const interface::Amount &amount() const override { return *amount_; }
 
       ModelType *copy() const override {
-        iroha::protocol::Command command;
-        *command.mutable_add_asset_quantity() = *add_asset_quantity_;
-        return new AddAssetQuantity(std::move(command));
+        return new AddAssetQuantity(iroha::protocol::Command(*command_));
       }
 
      private:
-      // ----------------------------| private API |----------------------------
-      explicit AddAssetQuantity(RefAddAssetQuantity &&ref)
-          : add_asset_quantity_(std::move(ref)), amount_([this] {
-              return proto::Amount(this->add_asset_quantity_->amount());
-            }) {}
-
       // ------------------------------| fields |-------------------------------
 
       // proto
-      RefAddAssetQuantity add_asset_quantity_;
-
-      template <typename Value>
-      using Lazy = detail::LazyInitializer<Value>;
+      detail::ReferenceHolder<iroha::protocol::Command> command_;
 
       // lazy
-      Lazy<proto::Amount> amount_;
+      const detail::LazyInitializer<iroha::protocol::AddAssetQuantity>
```

Comment:
Use `using` keyword for improving readability.

Reply:
It isn't reuse. It will only reduce readability